src/jdk.hotspot.agent/linux/native/libsaproc/libproc.h

```diff
@@ -80,7 +80,7 @@ int get_num_threads(struct ps_prochandle* ph);
 lwpid_t get_lwp_id(struct ps_prochandle* ph, int index);
 
 // get regs for a given lwp
-bool get_lwp_regs(struct ps_prochandle* ph, lwpid_t lid, struct user_regs_struct* regs);
```

Comment:
`jdk.hotspot.agent` component is for offline core investigation. It would require significant efforts, so likely should  be postponed. We can disable building the agent at all in makefiles: make/lib/Lib-jdk.hotspot.agent.gmk. Next line can be made conditional
```
TARGETS += $(BUILD_LIBSA)
```

(There is handy `isTargetCpu` function, you can see examples for `isTargetOs` in the file)

Reply:
Disabled the agent. And reverted all changes in src/jdk.hotspot.agent folder